README.md

```diff
@@ -3,10 +3,16 @@
 A zoom and pan plugin for Chart.js. Currently requires Chart.js >= 2.6.0
 
 Panning can be done via the mouse or with a finger.
-Zooming is done via the mouse wheel or via a pinch gesture. [Hammer JS](http://hammerjs.github.io/) is used for gesture recognition.
+Zooming is done via the mouse wheel or via a pinch gesture. [Hammer.js](http://hammerjs.github.io/) is used for gesture recognition.
 
 [Live Codepen Demo](http://codepen.io/pen/PGabEK)
 
+## Installation
+
+Run `npm install --save chartjs-plugin-zoom` to install with `npm`.
+
+If including via a `<script>` tag, make sure to include `Hammer.js` as well.
+
```

Comment:
I would add an example:

```
If including via a `<script>` tag, make sure to include `Hammer.js` as well:

```html
<script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
<script src="https://cdn.jsdelivr.net/npm/hammerjs@2.0.8"></script>
<script src="https://cdn.jsdelivr.net/npm/chartjs-plugin-zoom@0.7.0"></script>
```

Hammer doesn't need the extra path to the min file (`/hammer.min.js`). Until Chart.js 2.8, the path to the dist file was required (`chart.js@2.7.3/dist/Chart.min.js`) but it's not anymore the case since we changed [entry points](https://github.com/chartjs/Chart.js/blob/master/package.json#L7).

We may want the same for this plugin:

```javascript
"jsdelivr": "dist/chartjs-plugin-zoom.min.js",
"unpkg": "dist/chartjs-plugin-zoom.min.js",
"main": "dist/chartjs-plugin-zoom.js",
```